libs/coin-evm/src/abis/erc1155.abi.json

```diff
@@ -0,0 +1,314 @@
+[
```

Comment:
Am a noob, i can't understand this file usage, can you explain ? or document somewhere (maybe it's already done)

Reply:
https://solidity-fr.readthedocs.io/fr/latest/abi-spec.html
I would like to avoid documentating things that are "essentials" of how an EVM blockchain is working to avoid polluting with too much information.

I think the usage is pretty clear when you see the 
<img width="746" alt="Screenshot 2023-06-26 at 11 01 08" src="https://github.com/LedgerHQ/ledger-live/assets/44363395/52cf1622-eca7-49f6-b3e6-cc80c415d304">
method using the file.